app/views/static/release_notes.html.erb

```diff
@@ -1,6 +1,20 @@
 <div class="container">
   <h1><strong>Poznámky k zmenám funkcionality a štruktúry dát</strong></h1>
   <div class="long-text">
+    <h2 class="anchorable">Zmena k 3. 12. 2020</h2>
+    <h3>Autoform API</h3>
```

Comment:
zmeny na Autoforme tu nedokumentujeme, toto by malo byt len pre Datahub

Reply:
@jsuchal

Reply:
@pavolzbell nie, toto bolo od zaciatku myslene k vsetkym veciam. Nevidim dovod to drolit.

Reply:
@jsuchal dobre ale nepridame odkaz na tieto zmeny aj sem? https://ekosystem.slovensko.digital/sluzby/autoform/integracny-manual#api

Reply:
Pridajme.